lib/completions/dialects/claude.rb

```diff
@@ -17,7 +17,9 @@ def translate(generic_prompt)
 
           claude_prompt << "#{generic_prompt[:post_insts]}\n" if generic_prompt[:post_insts]
 
-          claude_prompt << "Assistant:\n"
+          claude_prompt << "Assistant:"
+          claude_prompt << " #{generic_prompt[:final_insts]}:" if generic_prompt[:final_insts]
```

Comment:
Why are final instructions here under Assistant? Should the lines be swapped in order?

Reply:
This is what we talked the other day - Injecting the first sentence of the assistant response gives you a  free boost

Reply:
Ohhh it works well with Claude? That's awesome!

Reply:
yeah works super well we should add it everywhere ... saves us from so many annoyances

Reply:
Yup, and if you are using XML tags to delimit the output, you can even put a small blurb + the opening tag to ensure a consistent reply from the bot.